analysis_scripts/analyze_mention_v_citation/mention_v_citation.md

```diff
@@ -0,0 +1,1080 @@
+    ## |:-----------------|----------------------:|---------------------:|-------------:|-------------:|-------------:|
+    ## |varieties         |              0.0164948|             0.0659794|     0.2500000|           162|           134|
+    ## |drought           |              0.0096154|             0.0312500|     0.3076923|           122|            79|
+    ## |soils             |              0.0141176|             0.0458824|     0.3076923|           125|            72|
+    ## |rice              |              0.0184941|             0.0594452|     0.3111111|           209|           215|
+    ## |farmers           |              0.0257689|             0.0681629|     0.3780488|           378|           237|
+    ## |crops             |              0.0222997|             0.0536585|     0.4155844|           529|           224|
+    ## |grain             |              0.0161290|             0.0376344|     0.4285714|            45|            23|
+    ## |reductions        |              0.0112994|             0.0240113|     0.4705882|           114|            37|
+    ## |yields            |              0.0231481|             0.0474537|     0.4878049|           171|            72|
+    ## |environmentalists |              0.0107143|             0.0196429|     0.5454545|            91|            19|
+    ## |villages          |              0.0204082|             0.0349854|     0.5833333|            99|            59|
+    ## |crop              |              0.0222694|             0.0381760|     0.5833333|           298|           147|
+    ## |tonnes            |              0.0169154|             0.0283582|     0.5964912|           374|           114|
+    ## |coal              |              0.0149068|             0.0248447|     0.6000000|           294|           131|
+    ## |agricultural      |              0.0284404|             0.0458716|     0.6200000|           384|           181|
+
+``` r
+# now take the top and bottom
+compare_freq = compare_freq[order(compare_freq$compare_ratio, decreasing=T),]
+compare_freq_extreme = compare_freq[c(1:15,(nrow(compare_freq)-14):nrow(compare_freq)),]
+compare_freq_extreme$word_type = c(rep("Citation", 15), rep("Mention", 15))
+
+# show the enrichment
+compare_freq_extreme$word = factor(compare_freq_extreme$word, 
+                                      levels = compare_freq_extreme$word)
+ggplot(compare_freq_extreme, aes(x=log10(compare_freq_extreme$compare_ratio), 
+                                 y=as.factor(compare_freq_extreme$word),
+                                 fill=word_type)) +
+    geom_bar(stat="identity") + theme_bw() + 
+    ylab("Words") + xlab("log10 Ratio Citation : Mention Frequencies") + 
+    ggtitle("log10 Ratio Citation : Mention Frequencies for most extreme words, normalized by country") + 
+    scale_fill_brewer(palette="Set2")
+```
+
+<img src="mention_v_citation_files/figure-markdown_github/calc_word_freq_diff-1.png" style="display: block; margin: auto;" />
```

Comment:
So the countries that have M>C (philippines, colombia, mexico, india) are not "UN identified industrialized countries", I'm still not 100% sure what goes into this definition, so this will need more clarification